web/src/components/tasks/cigarette-license/CigaretteLicenseStepOne.tsx

```diff
@@ -0,0 +1,42 @@
+import { Content } from "@/components/Content";
+import { HorizontalLine } from "@/components/HorizontalLine";
+import { Alert } from "@/components/njwds-extended/Alert";
+import { CtaContainer } from "@/components/njwds-extended/cta/CtaContainer";
+import { LiveChatHelpButton } from "@/components/njwds-extended/LiveChatHelpButton";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
+import { ActionBarLayout } from "@/components/njwds-layout/ActionBarLayout";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { ReactElement } from "react";
+
+interface Props {
+  setStepIndex: (step: number) => void;
+}
+
+export const CigaretteLicenseStepOne = (props: Props): ReactElement => {
+  const { Config } = useConfig();
+
+  return (
+    <>
+      <Alert variant={"info"}>
+        <Content>{Config.cigaretteLicenseStep1.alert}</Content>
+      </Alert>
+      <Content>{Config.cigaretteLicenseStep1.content}</Content>
+      <HorizontalLine />
+      <span className="h5-styling">{Config.cigaretteLicenseShared.issuingAgencyLabelText}: </span>
+      <span className="h6-styling">{Config.cigaretteLicenseShared.issuingAgencyText}</span>
```

Comment:
[pebble] I think we can simplify this.

This first step is essentially a single task screen. Given that's a pretty common pattern throughout the application, I think there are some currently established components that would allow us to just pass in the content and have the task screen logic render things for us without manually defining this.

This may take a little bit of reworking what's in the CMS, but this will have the added benefit of giving the content team complete freedom to modify this step.

As we move forward this next year we want to reduce the number of touchpoints where development is required to modify the application.